tst/EnergyPlus/unit/AirLoopHVACDOAS.unit.cc

```diff
@@ -4304,4 +4305,161 @@ TEST_F(EnergyPlusFixture, AirLoopHVACDOAS_TestOACompOutletNodeIndex)
     EXPECT_EQ(state->dataAirLoop->OutsideAirSys(1).OutletNodeNum(2), 23);
 }
 
+TEST_F(EnergyPlusFixture, AirLoopHVACDOAS_TestGetDesignDayConditions)
```

Comment:
Unit test provides 100% coverage of GetDesignDayConditions member function and also tests 2 DOAS systems to make sure 2nd finds sizing data.